mods/ts/weapons/healweapons.yaml

```diff
@@ -20,3 +23,5 @@ Repair:
 	ValidTargets: Repair
 	Warhead@1Dam: TargetDamage
 		ValidTargets: Repair
+	Warhead@2Eff: FlashTargetsInRadius
+		ValidTargets: Vehicle
```

Comment:
These don't seem right. Should they have matching ValidTargets?

Reply:
Not really, unless I reorder the warheads. Repair is condition based target for units with less than full hp. If first warhead heals to full then the second will not be valid